packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -162,6 +168,7 @@ class DropdownMenu<T> extends StatefulWidget {
     this.focusNode,
     this.requestFocusOnTap,
     this.expandedInsets,
+    this.filterCallback,
```

Comment:
Would it be possible make this default to `filter` or some wrapper of `filter`, rather than having to check if it's null later?

Reply:
If [filterCallback] is null, there is a default filter built-in that's used. I just followed the same structure searchCallback has. If I change this one and add a wrapper on that filter and attach it as default value, then I should change search as well to keep the code consistent. 

I would leave it as it is for now and maybe have an improvement PR that address that and maybe some simplification too

Reply:
Ok agreed, it makes sense to do this the same way as searchCallback for now.